### ISSUE TITLE
Importing POST requests with empty POST data fails

### ISSUE THREAD

**Issue by dotancohen** [a85/POSTMan-Chrome-Extension#4018]
## App Details:
```
Postman for Linux
Version 5.3.2
linux 3.10.0-693.11.1.el7.x86_64 / x64
```

## Issue Report:

1. Click "Import" button near upper-left corner of window.
2. Click "Paste Raw Text" button.
3. Paste the following request: `curl "http://simplesniff.com/" --data ""`
4. Click "Import" at bottom of dialogue.

**What happens:** Postman presents an error message:
`Error while importing Curl: TypeError: Invalid format for cURL.`

**Expected behaviour:** Import should work.

**Comment by prashantagarwal** [comment]
@dotancohen Thanks for reporting this. We were able to reproduce this. I will update the ticket once we release the fix.

**Comment by abhijitkane** [comment]
@dotancohen This should be fixed in the latest Postman version - 7.2+
